### ISSUE TITLE
Is there a way to use stack ?

### ISSUE THREAD

**Issue by Stargateur** [fussybeaver/bollard#167]
I would like to use https://docs.docker.com/engine/reference/commandline/stack_deploy/ but I don't see in https://docs.docker.com/engine/api/v1.41/

**Comment by fussybeaver** [comment]
It's not implemented at the moment, but looks relatively straightforward to add.

**Comment by Stargateur** [comment]
would avoid me to use `Command` so it would be nice.

**Comment by fussybeaver** [comment]
:thinking: Initially, I thought this was about adding an API endpoint that was missing, but looking at the CLI code, https://github.com/thaJeztah/cli/blob/master/cli/command/stack/swarm/deploy_composefile.go#L179-L267 it looks like they're just parsing the docker-compose file and calling the docker Service API.

**Comment by Stargateur** [comment]
yes I read some about it some people say docker didn't include this in the API to force user to use service like scaleway AWS etc. I don't know much about this, but docker compose exist since some years and they still didn't add it.
